### ISSUE TITLE
Design Discussion

### ISSUE THREAD

**Issue by ef4** [ryannpierce/open-source-branding#1]
Thanks for your work on this, and for offering to collaborate further. If you're interested in experimenting with some additional iterations, I would love to try to solve any of these design challenges.
- fitting in with the aesthetics of the wider Ember community, which runs toward playful / cute design. _I'm not asking for a Tomster, as Tilde is rather protective of Tomster, and while it may be possible to get a Liquid Fire Tomster, I'm pretty sure they would want to use their own designer to control the process_.
- establishing a color palette that's warm and friendly
- showing examples of how these designs can look in context on http://ef4.github.io/liquid-fire/
- iterating on the mark to try to produce a simplified version. It doesn't necessarily need to convey the whole story by itself -- a recognizable mark that only conveyed "fire" may be just as good as one that also tries to be liquid.
- the name itself doesn't need to have a literal meaning. It's a play on two synonyms: we build _fluid_ user experiences with _Ember_. If I had to pick some actual imagery, I think of volcanos or falling meteors.

**Comment by ryannmack** [comment]
I am just seeing this. Awesome direction @ef4, I'm going to work through some ideas. Thanks.

**Comment by ryannmack** [comment]
Here is a sketch of an idea I have been working on for a friendlier Liquid Fire logo, @ef4. In the final I will have softer/thinner edge lines on the face and more details on the meteor and the fire surrounding it. 

![liquid-fire-sketch](https://cloud.githubusercontent.com/assets/4509702/9285613/42fad9b2-42ad-11e5-8530-b2dc8fd8d4b5.jpg)

This is a rough idea for the color palette. I want to use purples vs black and grays for the darker areas. This will help with the warm and friendly feeling too.
![liquid-fire-colors](https://cloud.githubusercontent.com/assets/4509702/9285614/42fc89c4-42ad-11e5-977f-5e2d6a36dd92.png)

I'd like to hear your thoughts on this concept. Let me know if there are other ideas you'd like me to explore.

**Comment by ryannmack** [comment]
@ef4, here is a work in progress of the logo. Next I will be working on adding detail to the fire.

![liquid-fire-wip](https://cloud.githubusercontent.com/assets/4509702/9390652/08771eb0-4738-11e5-8d32-a063f8c8936e.jpg)

I have also tweaked the color palette.

![liquid-fire-colors-2](https://cloud.githubusercontent.com/assets/4509702/9390661/1691bd52-4738-11e5-8180-3e03ca916940.jpg)

I'd like to hear your thoughts on my progress.

**Comment by ef4** [comment]
Nice! Liking this direction. Great colors too.

**Comment by tim-evans** [comment]
Having a tomster that's a metalworker / welder would be interesting. (Or in a volcano-type heat suit)

**Comment by tim-evans** [comment]
By the way, I love this design. The comet / meteor is cute!

**Comment by ryannmack** [comment]
@tim-evans Thanks for the compliment, I can't wait to share what he looks like with flame details! 

A welding Tomster would be sweet but Tilde has <a href="http://emberjs.com/tomster/faq/#own">strong feelings</a> against using their mascot.

**Comment by tim-evans** [comment]
Agreed. I feel that those strong feelings go against the fact that liquid fire is an official addon.

**Comment by ef4** [comment]
I don't think it would be a problem getting an official liquid-fire Tomster, if we want one. I just didn't want to block @ryannpierce if she wants to dive in and do a whole design.

@ryannpierce, if you'd be excited about collaborating with @wifelette to integrate a Tomster, I'd be happy to facilitate. Either way is fine with me.

**Comment by ryannmack** [comment]
@ef4 I'd be thrilled to work with the official Tomster if we had their blessing!

**Comment by wifelette** [comment]
This is a super delayed reaction, but better late than never :stuck_out_tongue: 

I've been working on a grand-unifying-theme option for the official sub-projects, so I'm holding off on any Tomster stuff for them until that's sorted. The Ember Twiddle folks are also hoping for / waiting on a Tomster.

Hope to have news in early January!

**Comment by ryannmack** [comment]
Thanks for the update @wifelette. I'm excited to see the unifying theme!
